tangelo/linq/target/target_qiskit.py

```diff
@@ -142,28 +151,28 @@ def run_and_measure_one_shot(backend, translated_circuit):
         elif desired_meas_result is not None:
```

Comment:
Comment above, about the chunks etc (see other comment). That will help future readers.